### PR TITLE
Add a section introducing color helpers in Widget library.

### PR DESCRIPTION
**Pull request by seonmej**

### CODE REVIEW

File: ApplicationDeveloperGuide/UI/Widgets/widget.rst [hunk 1]
```diff
@@ -38,6 +38,23 @@ Containers:
 - ``OverlapContainer``: lays out any number of children by stacking them.
 - ``Canvas``: lays out any number of children freely.
 
+Color Utilities
+---------------
+
+The widget library offers some color utilities.
+
+The ``ColorHelper`` is helpful for exploding colors into components, alpha, red, blue, green and building back a color from components.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/UI/Widgets/widget.rst:46]: Please add links to javadoc

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

File: ApplicationDeveloperGuide/UI/Widgets/widget.rst (hunk 1 above)
Comment (seonmej) [reply ApplicationDeveloperGuide/UI/Widgets/widget.rst]: fixed

File: ApplicationDeveloperGuide/UI/Widgets/widget.rst [hunk 2]
```diff
@@ -38,6 +38,23 @@ Containers:
...
+
+The ``GradientHelper`` is able to blend two colors and can create a gradient from 2 colors.
```
[the remark this answers is not in the document]
Comment (seonmej) [reply ApplicationDeveloperGuide/UI/Widgets/widget.rst]: fixed

File: ApplicationDeveloperGuide/UI/Widgets/widget.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (seonmej) [reply ApplicationDeveloperGuide/UI/Widgets/widget.rst]: fixed

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**